plugins/modules/azure_rm_virtualnetworkpeering.py

```diff
@@ -330,6 +352,41 @@ def get_vnet(self, resource_group, vnet_name):
             return results
         return False
 
+    def sync_vnet_peering(self):
+        '''
+        Creates or Update Azure Virtual Network Peering.
+
+        :return: deserialized Azure Virtual Network Peering instance state dictionary
+        '''
+        self.log("Creating or Updating the Azure Virtual Network Peering {0}".format(self.name))
+
+        vnet_id = format_resource_id(self.virtual_network['name'],
+                                     self.subscription_id,
+                                     'Microsoft.Network',
+                                     'virtualNetworks',
+                                     self.virtual_network['resource_group'])
+        peering = self.network_models.VirtualNetworkPeering(
+            id=vnet_id,
+            name=self.name,
+            remote_virtual_network=self.network_models.SubResource(id=self.remote_virtual_network),
+            allow_virtual_network_access=self.allow_virtual_network_access,
+            allow_gateway_transit=self.allow_gateway_transit,
+            allow_forwarded_traffic=self.allow_forwarded_traffic,
+            use_remote_gateways=self.use_remote_gateways
+            )
```

Comment:
closing bracket does not match indentation of opening bracket's line
```suggestion
        )
```

Reply:
Removed whitespace